Lib/idlelib/editor.py

```diff
@@ -999,6 +1037,8 @@ def _close(self):
         if self.color:
             self.color.close(False)
             self.color = None
+        # Allow code context to close its text.after calls.
```

Comment:
How does unbinding the event allow the code context to do anything?

Also this is surprising to find given the title of this PR. If kept, it should be mentioned in the description and in the final commit message as an additional fix.